build.gradle

```diff
@@ -19,7 +19,7 @@ buildscript {
         junit_version = '4.13'
         maven_publish_version = '3.6.2'
         mockito_core_version = '2.24.5'
-        roboelectric_core_version = '4.3.1'
+        robolectric_core_version = '4.5.1'
```

Comment:
I needed this new version to be able to run the Kotlin tests locally, for inscrutable reasons.